### ISSUE TITLE
Incorrect WAI-ARIA state on Actions Button in Content Folder View

### ISSUE THREAD

**Issue by ma10** [plone/Products.CMFPlone#867]
The initial state of the button the right-most column of the folder content views seems to be incorrect.

When you first load a folder content view, the screen readers report the menu is expanded, while it is actually collapsed.

If you hit Enter on it once, it is reported, again. to be expanded.  And hitting Enter again on it causes screen readers to correctly report it to be collapsed.  After that, the state of the button (menu) is correctly reported.

Tested Environments:

* Firefox 40.03 (32bit) with JAWS 14.0 on Windows
* Firefox 40.03 (32bit) with NVDA 2015.3 on Windows
* Google Chrome 45.0.2454.85 m (64-bit) with JAWS 14.0 on Windows
* Google Chrome 45.0.2454.85 m (64-bit) with NVDA 2015.3 on Windows

**Comment by davisagli** [comment]
For clarity: this seems to be about the "Filter" button, which doesn't have `aria-hidden="true"` until it has been opened and closed once.

**Comment by jensens** [comment]
Closing as outdated (4 years old). About incorrect ARIA state on Actions button in old Mockup folder contents. Folder contents rewritten for Plone 6.
If you think this issue is still relevant please reopen it. Thank you for the original report!
